tests/conftest.py

```diff
@@ -1,6 +1,12 @@
 import pytest
 from airflow.models import DAG
-from airflow.operators.empty import EmptyOperator
+
+try:
+    from airflow.operators.empty import EmptyOperator
+except ModuleNotFoundError:
+    # For older versions of Airflow which do not contain the EmptyOperator.
```

Comment:
Worth stating which version of Airflow, so we can remove this check once we stop supporting that Airflow version

Reply:
yes, stated the version.